### ISSUE TITLE
Track titles with multiple suffixes

### ISSUE THREAD

**Issue by djmattyg007** [web-scrobbler/metadata-filter#103]
I've come across a track title on Spotify that is literally `Fa Fa Fa - (Album Version)`. The `removeVersion` filter correctly strips `(Album Version)`, leaving it as `Fa Fa Fa - `. However, I'm not aware of a filter that will strip the ` - ` at present.

It's also worth noting that the pre-defined Spotify filter doesn't currently strip version suffixes like this. I'd like to submit a PR to add it, unless you have an objection.

**Comment by djmattyg007** [comment]
Actually, I remembered the `TRIM_SYMBOLS_FILTER_RULES` set of filters, which looks like it would be sufficient. Perhaps these should both be added at to the predefined Spotify filter at the same time?

**Comment by alexesprit** [comment]
Yes, it could be reused in Spotify filter to process such cases.
